src/CameraControls.ts

```diff
@@ -1138,57 +1157,67 @@ export class CameraControls extends EventDispatcher {
 
 		};
 
-		const lockPointer = (): void => {
+		this.lockPointer = (): void => {
 
 			if ( ! this._enabled || ! this._domElement ) return;
 
 			this.cancel();
 
 			// Element.requestPointerLock is allowed to happen without any pointer active - create a faux one for compatibility with controls
-			const pointer: PointerInput = {
-				pointerId: 1,
+			this._lockedPointer = {
+				pointerId: - 1,
 				clientX: 0,
 				clientY: 0,
 				deltaX: 0,
 				deltaY: 0,
-				mouseButton: null
+				mouseButton: null,
 			};
```

Comment:
save as `this._lockedPointer` while pointer-lock is active.
then
- `this._lockedPointer` should not be removed from `activePointers`
- if `this._lockedPointer` is not `null`, that means pointer-lock is active.